ldap_probe/models.py

```diff
@@ -80,7 +79,7 @@ class LdapProbeLogAnonBindManager(models.Manager):
     for most of `AD` controllers.
     By design LDAP probe data collected from these `AD` controllers will
     present with :attr:`LdapProbeLog.elapsed_anon_bind` values that are not
-    0.
+    null (`None` in `Python` speak).
```

Comment:
Same as above, double backticks
```suggestion
    null (``None`` in `Python` speak).
```